src/webui/prefjson.cpp

```diff
@@ -114,6 +114,7 @@ QByteArray prefjson::getPreferences()
     data["ip_filter_enabled"] = session->isIPFilteringEnabled();
     data["ip_filter_path"] = Utils::Fs::toNativePath(session->IPFilterFile());
     data["ip_filter_trackers"] = session->isTrackerFilteringEnabled();
+    data["banned_ip_list"] = session->getBannedIPList().join("\n");
```

Comment:
I got curios and roughly counted getSmth() style function declarations. `./src/base/preferences.h` have 71 such declarations and whole project nearly 179. Looks like the train has left the station.